apps/andi/priv/repo/migrations/20220613202349_rename_submission_status.exs

```diff
@@ -0,0 +1,7 @@
+defmodule Andi.Repo.Migrations.RenameSubmissionStatus do
+  use Ecto.Migration
+
+  def change do
+    rename table(:ingestions), :submission_status, to: :submissionStatus
```

Comment:
Since this isn't deployed anywhere yet, do we need to have this name change migration? Could we just update the original migration to be named submissionStatus?

Reply:
I believe this was deployed to dev through a side-branch. I thought it would be harmless to treat it like a live schema, worst case ecto goes through one additional step than it strictly speaking needs to.